app/overrides/satellite_name_override.rb

```diff
@@ -37,7 +37,7 @@
                      :text          =>  "    <div class=\"stats-well\"><h4><%= _(\"Support\") %></h4> <p>Visit the <%= link_to _('Customer Portal'), \"https://access.redhat.com/\",
                                        :rel => \"external\" %> to get support, find solutions to common questions, and more.</p><h6><%= _(\"Documentation\") %></h6>
                                         <ul>
-                                        <li><%= link_to _('Complete Product Documentation for Red Hat Satellite'),\"https://access.redhat.com/documentation/en/red_hat_satellite/#{ForemanThemeSatellite::SATELLITE_SHORT_VERSION}\", :rel => \"external\" %></li>
+                                        <li><%= link_to _('Complete Product Documentation for Red Hat Satellite'),\"https://docs.redhat.com/documentation/en/red_hat_satellite/#{ForemanThemeSatellite::SATELLITE_SHORT_VERSION}\", :rel => \"external\" %></li>
```

Comment:
```suggestion
                                        <li><%= link_to _('Complete Product Documentation for Red Hat Satellite'),\"#{ForemanThemeSatellite.documentation_root}\", :rel => \"external\" %></li>
```

And we can do it for all the `docs.redhat.com` links in this PR

Reply:
`documentation_root` includes `html` so it isn't the same:
https://github.com/RedHatSatellite/foreman_theme_satellite/blob/bf42fb119af723df5080e9b5d0f73fd8c483e017/lib/foreman_theme_satellite/engine.rb#L149-L153

That's why I didn't change it.